Src/MoneyFox.Win/MoneyFox.Win/Converter/LastDayOfMonthVisibilityConverter.cs

```diff
@@ -0,0 +1,19 @@
+namespace MoneyFox.Win.Converter;
+
+using Core.Aggregates.Payments;
+using Core._Pending_.Common.Helpers;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Data;
+using System;
+
+public class LastDayOfMonthVisibilityConverter : IValueConverter
```

Comment:
I think I would rather make a bool property in the viewmodel for that and bind the visibility to that.

Reply:
@NPadrutt - That's an excellent idea. I removed the stand-alone iValueConverter class and moved the visibility logic into the ViewModel. Much cleaner. Thanks for the tip!